atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -299,7 +305,7 @@ public void hostIsAutomaticallyRemovedOnStartup() {
         setCassandraServersTo(CASS_SERVER_1);
 
         createClientPool();
-        assertThat(poolServers).containsExactlyInAnyOrder(CASS_SERVER_1);
+        assertThat(poolServers.keySet()).containsExactlyInAnyOrder(CASS_SERVER_1);
```

Comment:
I actually don't know where our preference generally lies, but there does exist a containsOnlyKeys method on the MapAssert so you don't need to keySet

Reply:
`       assertThatMap(poolServers).containsOnlyKeys(CASS_SERVER_1);` is much more eloquent imo